resume-buddy/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -35,29 +34,33 @@
 @WebServlet("/login")
 public class LoginServlet extends HttpServlet {
 
-  /* TODO: Find a way for the user type to be passed to the CommentsServlet from JavaScript
-   * (instead of being stored here as a static variable) */
-  private static UserType userType;
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
     UserService userService = UserServiceFactory.getUserService();
-    boolean isValidUser;
+    boolean status = userService.isUserLoggedIn();
```

Comment:
Nit: You don't need this variable, just use `userService.isUserLoggedIn()` for readability.